app/extensions/ember-table/template.hbs

```diff
@@ -0,0 +1,10 @@
+<div class="ui basic less padding segment x-scrollable">
```

Comment:
@CosmicCoder96 Change it to this, Zero Overflow ! 😄 
```js
<div class="ui segment column sixteen wide inner-table-wrapper x-scrollable">
  <table class="ui very basic unstackable table">
    {{yield (hash
              api=api
              head=(component "ember-thead" api=api)
              body=(component "ember-tbody" api=api)
              foot=(component "ember-tfoot" api=api)
            )}}
  </table>
</div>
```

Reply:
doesn't fix the core of the problem. It's the non-reducing text. @kushthedude The non basic segment makes it worse.
![image](https://user-images.githubusercontent.com/17252805/60599815-b78ef780-9dcc-11e9-8518-2bf406fc7793.png)

Reply:
@CosmicCoder96 for x scroll within the table, use the functionality provided by ember-table

Reply:
@CosmicCoder96 I know it, But previously we handled it by illusions, The very basic table just removes all the lines and the outer segment which is wrapped acts like table border.
And it looks like this now 
![image](https://user-images.githubusercontent.com/44091822/60599990-076dbe80-9dcd-11e9-846d-918a427a50d8.png)

Reply:
@niranjan94 checking it out.